Multisig/Data/Services/Utils/HTTPClient.swift

```diff
@@ -35,6 +35,13 @@ class HTTPClient {
         self.logger = logger
     }
 
+    fileprivate enum UnexpectedError: LoggableError {
```

Comment:
All good, but I prefer to find a way to define all those LoggableError enums in one place instead of define each couple values in separate file

Reply:
I see. I think, good middle ground would be to put all such errors in the "Cross-layer" directory

Reply:
something like so, because I think later we might add more, and keeping them dispersed make it hard to maintain

Reply:
ok, done